sycl/test-e2e/Matrix/joint_matrix_bfloat16_colmajorA_colmajorB.cpp

```diff
@@ -5,6 +5,8 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+// VNNI transform and sub-group size 32 are not supported yet on DG2 by IGC
```

Comment:
sub-group size 32 part is not relevant for this test.
```suggestion
// VNNI transform is not supported yet on DG2 by IGC
```